scripts/create_bridges.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Service to create bridges
+After=libvirtd.service
```

Comment:
This should run before libvirtd.

We should also make this run after the networks target.